tests/ui/test_validation_ui.py

```diff
@@ -40,6 +42,7 @@ def test_dashboard_validation_ui(self, setup_ui):
     @tier1
     @skipif_ui_not_support("validation")
     @skipif_ocs_version("<4.9")
+    @blue_squad
```

Comment:
This is not a blue squad test case.

Reply:
@ebenahar any thoughts on which squad could be a better fit? 
This is a basic test that checks if the dashboard works. so it is not blue squad as per our definitions. But, it doesn't fall under any other category as well. 
Maybe purple squad?

Reply:
indeed no squad has this area by definition. 
@keesturam can you keep it under black for now (even though we would like to deprecate this squad), until we define which squad will take this area